tutorials/filtering-prometheus-metrics.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Filtering Prometheus metrics
+description: Learn how to setup filtering Prometheus metrics on the Pod side
+author: ganochenkodg
+tags: Prometheus, metrics
+date_published: 2020-12-07
+---
+
+Ganochenko Dmitrii
+
+<p style="background-color:#D9EFFC;"><i>Contributed by the Google Cloud community. Not official Google documentation.</i></p>
```

Comment:
Please follow the template for submissions from Google employees.

Reply:
Fixed, could you check it again pls